src/client/datascience/notebook/intellisense/intellisenseProvider.ts

```diff
@@ -175,4 +185,18 @@ export class IntellisenseProvider implements IExtensionSyncActivationService {
 
         return id ? this.servers.get(id) : undefined;
     }
+
+    private onDidChangeConfiguration(event: ConfigurationChangeEvent) {
+        if (
+            event.affectsConfiguration('jupyter.pylanceHandlesNotebooks') ||
+            event.affectsConfiguration('python.languageServer')
```

Comment:
This probably doesn't matter, but feasibly someone could change from "Pylance" to "Default" or vice versa, in which case the config changes from VS Code's view, but under the hood, the Python extension does nothing.

I doubt it really matters, but thought I'd mention.